core-build/gulp-core-build/src/tasks/JestTask.ts

```diff
@@ -136,7 +141,16 @@ export class JestTask extends GulpTask<IJestConfig> {
       moduleDirectories: this.taskConfig.moduleDirectories ?
         this.taskConfig.moduleDirectories :
         ['node_modules', this.buildConfig.libFolder],
-      reporters: [path.join(__dirname, 'JestReporter.js')],
+      reporters: [
+        [
+          path.join(__dirname, 'JestReporter.js'),
+          {
+            outputPath: path.join(this.buildConfig.tempFolder, 'jest-results'),
+            outputFilename: 'test-results.xml',
```

Comment:
Would it be easier to combine these two options and have the config provide a full path to the output file?